cmd/vault-manager/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"os"
+
+	"github.com/alecthomas/kingpin"
+
+	"k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" // this is required to auth against GCP
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/gocardless/theatre/cmd"
+	"github.com/gocardless/theatre/pkg/apis"
+	"github.com/gocardless/theatre/pkg/signals"
+	"github.com/gocardless/theatre/pkg/vault/envconsul"
+)
+
+var (
+	app                       = kingpin.New("vault-manager", "Manages vault.crd.gocardless.com resources").Version(Version)
+	namespace                 = app.Flag("namespace", "Kubernetes webhook service namespace").Default("theatre-system").String()
+	serviceName               = app.Flag("service-name", "Webhook service name").Default("theatre-vault-manager").String()
+	webhookName               = app.Flag("webhook-name", "Mutating webhook name").Default("theatre-vault").String()
+	webhookConfigMapName      = app.Flag("webhook-configmap-name", "Webhook configMap name containing theatre-envconsul configuration").Default("theatre-vault-manager-webhook").String()
```

Comment:
I commented in the other PR, I think we can and should remove this.

We shouldn't need any configuration, and it makes things much simpler if we can just take flags set by environment variables, for example.

https://github.com/gocardless/theatre/pull/79/files#r350658999

Reply:
Have removed this now and accept flags for `--theatre-image` and `--install-path`